client/src/components/overview/overview-components/Selector.jsx

```diff
@@ -33,14 +32,19 @@ const StyledOverviewStylesSection = styled.div`
 `;
 const StyleCircle = styled.div`
   align-items: center;
-  width:40px;
-  height:40px;
+  width:50px;
+  height:50px;
   overflow: hidden;
   border-radius: 50%;
   background-size: cover;
   display: flex;
   justify-content: flex-end;
   align-items: flex-start;
+  cursor: pointer;
+  transition: 0.3s;
+  &:hover {
+    box-shadow: 2px 2px 5px black;
+  }
 `;
```

Comment:
Style selector changes, just changing the size of the icons and the gaps between them